src/base/static/components/organisms/map-legend-panel.js

```diff
@@ -15,40 +18,26 @@ const MapLegendPanelContainer = styled("div")(props => ({
 const MapLegendPanel = props => {
   return (
     <MapLegendPanelContainer isThemed={props.isThemed}>
-      {props.config.title && <Header4>{props.config.title}</Header4>}
-      {props.config.description && (
-        <Paragraph>{props.config.description}</Paragraph>
-      )}
```

Comment:
I dropped support for including a title and description on the MapLegendPanel, which I think is only used on the oakland flavor